generator/common.py

```diff
@@ -1,6 +1,47 @@
-constants_path = "../constants.h"
-can_lib_c_path = "../Can_Library.c"
-can_lib_h_path = "../Can_Library.h"
-enum_segments_path = "../enum_segments.h"
+constants_path = '../constants.h'
+can_lib_c_path = '../Can_Library.c'
+can_lib_h_path = '../Can_Library.h'
+enum_segments_path = '../enum_segments.h'
+structs_path = '../structs.h'
 
-can_lib_c_base_path = "templates/Can_Library_BASE.c"
+can_lib_c_base_path = 'templates/Can_Library_BASE.c'
+
+def coord(*args):
```

Comment:
I definitely like the addition of this file. I think it also makes it easier for users of the library to figure out what names they should use based on the actual code and not just trusting the docs.